client/components/reader-full-post-header/style.scss

```diff
@@ -0,0 +1,99 @@
+.reader-full-post-header__title {
+	clear: none;
+	color: $gray-dark;
+	font-family: $serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	font-size: 20px;
+	font-weight: 700;
+	line-height: 1.3;
+	margin: 24px 0 8px;
+	max-width: 750px;
+
+	@include breakpoint( ">480px" ) {
+		font-size: 28px;
+		line-height: 1.4;
+	}
+
+	.reader-full-post-header__title-link,
+	.reader-full-post-header__title-link:hover {
+		color: $gray-dark;
+	}
+}
+
+.reader-full-post-header__meta {
+	font-size: 15px;
+	max-width: 750px;
+}
+
+.reader-full-post-header__date {
+	float: left;
+}
+
+.reader-full-post-header__date-link {
+	margin-right: 25px;
+}
+
+.reader-full-post-header__date-link,
+.reader-full-post-header__date-link:visited {
+	color: $gray;
+}
+
+.reader-full-post-header__date-link:hover {
+	color: darken( $gray, 20% );
+}
+
+.reader-full-post-header__tags {
+	display: inline-block;
+    width: calc(100% - 120px);
+
+	.gridicon {
+		fill: $gray;
+		float: left;
+		margin-right: 5px;
+		position: relative;
+			top: 4px;
+	}
+}
+
+.reader-full-post-header__tag-list {
+	color: $gray;
+	list-style: none;
+	margin-left: 20px;
+    padding: 0;
+    position: relative;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+
+    &::after {
+    	@include long-content-fade( $size: 35% );
+    }
+
+	li {
```

Comment:
For both the `li` and `a`? Do we need to add this to the CSS guidelines?

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md